src/index.ts

```diff
@@ -671,6 +671,102 @@ export interface JIT {
   extensions?: { [key: string]: string }
 }
 
+/**
+ * FormFieldIdentitySpace
+ */
+export interface FormFieldIdentitySpace {
+  id?: string
+  name?: string
+  code?: string
```

Comment:
this is a very different setup to normal identity spaces. can you please explain what is going on here?

Reply:
form field can be associated w/ formField, we just associate formField w/ identity using `ID` as foreign to pull in that info. the formField will use the `formField.identitySpace.code` to add identity to the the right-invocation payload

Reply:
I still don't understand your message. What is `id` and `name` for?